tests/test_calibrateImage.py

```diff
@@ -216,6 +232,8 @@ def test_handle_snaps(self):
             calibrate._handle_snaps([])
         with self.assertRaisesRegex(RuntimeError, "Can only process 1 or 2 snaps, not 3."):
             calibrate._handle_snaps(3*[self.exposure])
+        with self.assertRaisesRegex(RuntimeError, "must be either an afw Exposure"):
```

Comment:
I see an "either" without an "or" here. (Also, I know the guide doesn't say anything about error messages, but it'd be nice to have them as a complete sentence.)

Reply:
The error message itself (line 580 of the main code) is a complete sentence. This is testing that a regex (which in this case is just a partial text of the message) is included. There's not much point in fully writing out the message here, as checking the exact text of the message isn't as important as confirming that this particular RuntimeError was raised (as opposed to the other one, in the assert just above this).